doc/manuals/devel/cookbook.md

```diff
@@ -420,11 +424,9 @@ Finally, execute the functional test for `test_to_debug.test`:
 CB_MAX_TRIES=1 /path/to/testHarness.sh /path/to/test_to_debug.test
 ```
 
-The test will start execution using the contextBroker process under debugger. For instance, if you
-have set a breakpoint in a place traversed by the .test cases, the execution will stop there.
-
-Once you have ended, remember to restore the `brokerStart`, `brokerStop` and (maybe) `dbDrop` lines
```

Comment:
I wouldn't remove this line, people tends to forget to recover commented lines. I'd suggest a new formulation:

```
Once you have ended, remember to restore any commented line in the .test file (`dbDrop`, etc.)
```

Reply:
Seems to be already included in L402... I have missed in the  diff. NTC

Reply:
It's not removed, but mentioned a few lines up, where I explain that it may be a good ida to comment out 'dbDrop'